vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -94,17 +104,31 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         return data;
     }
 
+    private List<String> extractValues(IExcelSheetParser sheetParser, String range)
+    {
+        return sheetParser.getDataFromRange(range).stream().map(CellValue::getValue).collect(Collectors.toList());
+    }
+
     private List<String> extractDataFromAddresses(IExcelSheetParser sheetParser, String addresses)
     {
         return Stream.of(addresses.split(";")).map(sheetParser::getDataFromCell).collect(Collectors.toList());
     }
 
-    private String build(List<String> data, TableProperties properties)
+    private String build(List<String> headers, List<List<String>> data, TableProperties properties)
```

Comment:
you can use Collection interface instead of List and pass exactDataTable keys and values without collecting them into list